omnigibson/simulator.py

```diff
@@ -55,6 +67,16 @@
 
 m.OBJECT_GRAVEYARD_POS = (100.0, 100.0, 100.0)
 
+# List of prim paths that should still be on the stage after clearing the simulator.
+CLEAR_ALLOWED_PRIM_PATH_REGEXES = [
+    "/physicsScene",
+    "/World",
```

Comment:
This can be removed now. The stage will actually be fully cleared when we init a new stage.